adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/iOS/PlatformInformation.cs

```diff
@@ -70,7 +71,7 @@ public override string GetDeviceModel()
         public override string GetAssemblyFileVersionAttribute()
         {
             // TODO: Check if assembly file version can be read in iOS assembly as well or not. For now, we use assembly version instead.
-            return typeof(AdalIdHelper).GetTypeInfo().Assembly.GetName().Version.ToString();
+            return typeof(UriParamsHelper).GetTypeInfo().Assembly.GetName().Version.ToString();
```

Comment:
>typeof(UriParamsHelper).GetTypeInfo().Assembly.GetName().Version.ToString(); [](start = 19, length = 76)

IMO: should only be "calculated" once and then return the calculated value, this thing is wasting lots of CPU cycles